runtime/src/iree/modules/vmvx/module.c

```diff
@@ -689,11 +694,12 @@ IREE_VMVX_ABI_EXPORT(iree_vmvx_unpack, unpack, v) {
                            /*buffer_ref=*/args->out_ref,
                            /*offset=*/args->out_offset,
                            /*stride0=*/args->out_stride0,
-                           /*stride1=*/1,
+                           /*stride1=*/args->out_stride1,
```

Comment:
I can't comment on above lines. Should `/*stride1=*/1` be `args->in_stride1` in l.690?